source/ui/Components/Output/Console.qml

```diff
@@ -1,75 +1,73 @@
 import QtQuick 2.0
 import QtQuick 2.3
 import QtQuick.Controls 1.4
+import QtQuick.Controls.Styles 1.4
 
 Item {
     property int outputItemIndex: 2
     property int oldMode: 0
     property var oldAddress: 0
     id: it
 
-
-    Rectangle{
-        anchors.fill: parent
-        color: "black"
-
-        TextArea{
-            id: textarea
-            height: parent.height
-            width: parent.width
-            anchors.top: parent.top
-            anchors.left: parent.left
-
-            wrapMode: Text.Wrap
+    TextArea {
+        id: textarea
+        height: parent.height
+        width: parent.width
+        anchors.top: parent.top
+        anchors.left: parent.left
+        style: TextAreaStyle {
             textColor: "white"
-
-            readOnly: true
-            backgroundVisible: false
-            MouseArea{//Clears the Screen in pipelike mode
-                id: clearArea
-               anchors.fill:parent
-               visible: true
-               acceptedButtons: Qt.RightButton
-               onDoubleClicked: {
-                   if(outputComponent.getOutputItem(outputItemIndex)["textMode"]!==0){
-                       textarea.text = "";
-                   }
-               }
-            }
-            function clear(){
-                text = "";
+            backgroundColor: "black"
+        }
+        wrapMode: Text.Wrap
+        readOnly: true
+        backgroundVisible: true
+
+        //Clears the Screen in pipelike mode
+        MouseArea {
+            id: clearArea
+            anchors.fill:parent
+            visible: true
+            acceptedButtons: Qt.RightButton
+            onDoubleClicked: {
+                if (outputComponent.getOutputItem(outputItemIndex)["textMode"] !== 0) {
+                    textarea.clear();
+                }
             }
-
         }
+        function clear() {
+            text = "";
```

Comment:
Interesting: writing a `\0` to memory in pipe mode also clears the console. Is this intended behaviour or is it a bug?